src/crimson/osd/replicated_backend.cc

```diff
@@ -36,18 +38,21 @@ seastar::future<crimson::osd::acked_peers_t>
 ReplicatedBackend::_submit_transaction(std::set<pg_shard_t>&& pg_shards,
                                        const hobject_t& hoid,
                                        ceph::os::Transaction&& txn,
-                                       osd_reqid_t req_id,
+                                       OSDOpParams&& osd_op_p,
                                        epoch_t min_epoch, epoch_t map_epoch,
-                                       eversion_t ver)
+				       vector<pg_log_entry_t> logv)
 {
   const ceph_tid_t tid = next_txn_id++;
+  auto req_id = osd_op_p.req.get_reqid();
+  eversion_t last_complete=osd_op_p.last_complete;
```

Comment:
please add spaces around `=`. 

also, could add `const` before `eversion_t` to help reader to understand this variable does not change in its life cycle.